src/utilities/createArrayAttributes.tsx

```diff
@@ -0,0 +1,8 @@
+/**
+ * Create an array with a given number of elements or size
+ * @param numberOfElements
+ * @returns
+ */
+export const createArrayAttributes = (numberOfElements: number) => {
```

Comment:
for both of these utility files, you'll just want to export the function as default exports. If you only ever have 1 function/component in a file, always just use the default export